pkg/cmd/pulumi/package_add.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2024, Pulumi Corporation.
+//Copyright 2024, Pulumi Corporation.
```

Comment:
```suggestion
// Copyright 2024, Pulumi Corporation.
```
Changes like this just make `git log` harder to read